### PR TITLE
preseed.cfg.sd: support for debian installation on sdcard

### PR DESCRIPTION
**Pull request by pkonkol**

### CODE REVIEW

**Review by lwcislo — APPROVED** [review]

**Comment by lwcislo** [comment]
@miczyg1 It looks fine for me and if you could also take a look I'd appreciate.

**Comment by miczyg1** [comment]
@lwcislo it should be a copy of other preseeds with a disk changed to mmcblk0. AS I can see it has been done. If it installs that is ok. Do you have a log from robot?

**Comment by lwcislo** [comment]
@miczyg1 https://jira-3mdeb.atlassian.net/browse/PE-628?focusedCommentId=40644

**Comment by pkonkol** [comment]
@miczyg1 @lwcislo I've commited another preseed for devices with mcr

File: preseed.cfg.mcr.sd [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+d-i mirror/http/directory string /debian
+d-i mirror/http/proxy string
+d-i clock-setup/utc boolean true
+d-i clock-setup/ntp boolean true
+d-i time/zone string Europe/Warsaw
+d-i partman/mount_style select uuid
+d-i partman/confirm boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm_nooverwrite boolean true
+d-i partman-auto/disk string /dev/disk/by-id/usb-Multiple_Card_Reader_058F63666485-0:0
```
Comment (miczyg1) [inline preseed.cfg.mcr.sd:25]: Is this multiple card reader the same for apu1 and apu5?

Comment (pkonkol) [reply preseed.cfg.mcr.sd:25]: Checked on apu5: 
```
root@debian:~# ls -l /dev/disk/by-id/
total 0
lrwxrwxrwx 1 root root  9 Jun 18 16:51 usb-Multiple_Card_Reader_058F63666485-0:0 -> ../../sdc
```
It's the same

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**